README.md

```diff
@@ -225,6 +225,28 @@ To run a specific test, run `tox -e <testenv>`. Test environments include:
 
 [tox]: https://pypi.org/project/tox/
 
+## DICOM Standard Workarounds
```

Comment:
This is great; I'm sure it was a decent amount of work to put this together, so thank you!

Reply:
Hopefully it will convince some people, who may otherwise venture to "reinvent the wheel", to use this project instead.